src/pubsub/peer-streams.js

```diff
@@ -138,26 +140,26 @@ class PeerStreams extends EventEmitter {
   /**
    * Attach a raw outbound stream and setup a write stream
    *
-   * @param {Stream} stream
+   * @param {MuxedStream} stream
    * @returns {Promise<void>}
    */
   async attachOutboundStream (stream) {
     // If an outbound stream already exists,
     // gently close it
     const _prevStream = this.outboundStream
```

Comment:
In that case this line is no longer needed
```suggestion
```

Reply:
We need it for the check in the end of the function to only emit `stream:outbound` if the connection is new. We should only emit it when the outbound stream is ready, so this should be in the end, which means we need that auxiliary constant